### DISCUSSION TITLE
Shared albums

### DISCUSSION THREAD

**Discussion by smolesen** [photoprism/photoprism#3278] [Q&A]
Is there a smart way, to share multiple (50+) albums to the same "link"... seems like I have to select an album, select share and then paste the "key" into the share dialog, then deselect and move on the the next album....rather cumbersome. If I select multiple Albums, the share option is grayed out....

Also, is there a way to get an overview of which albums are shared via a "links"? 
TIA

/Søren

**Comment by graciousgrey** [comment]
Not yet, but both is planned.
